### ISSUE TITLE
Update numpy decoding

### ISSUE THREAD

**Issue by JeroenvO** [bennomeier/leCroyParser#6]
Hi,
reading binary data with numpy fromstring is deprecated: 
https://numpy.org/devdocs/reference/generated/numpy.fromstring.html

It should be changed to numpy.frombuffer.

**Comment by JeroenvO** [comment]
Maybe use the `count` and `offset` arguments instead of trimming the input data.

**Comment by JeroenvO** [comment]
To be precise:
```
__init__.py:144: DeprecationWarning: The binary mode of fromstring is deprecated, as it behaves surprisingly on unicode inputs. Use frombuffer instead
x = np.fromstring(self.file.read(length), self.endianness + formatSpecifier)[0]
```

**Comment by JeroenvO** [comment]
Fixed in #8

**Comment by bennomeier** [comment]
Thanks.
